tests/integration/components/bs-form-test.js

```diff
@@ -682,7 +699,15 @@ module('Integration | Component | bs-form', function(hooks) {
   });
 
   test('Yielded #isRejected is true if onSubmit action rejects', async function(assert) {
-    this.actions.submit = this.fake.rejects();
+    // tests fail by default on unhandled errors
+    let expectedError = new Error();
+    Ember.onerror = (error) => {
+      if (error !== expectedError) {
+        throw error;
+      }
+    };
```

Comment:
Shoudn't we restore the original `Ember.onerror` handler after the test? Or is it undefined by default?

Reply:
If I didn't missed something `Ember.onerror` is initialized with `undefined` for each test. I'm not sure if this is documented behavior and should be treated as public API. Could reset it for clarity if you prefer that.